cmd/tracee/cmd/man.go

```diff
@@ -125,21 +125,36 @@ var scopeCmd = &cobra.Command{
 
 // runManForFlag runs man for the specified flag name
 func runManForFlag(flagName string) error {
-	const manHelpPath = "./docs/man"
-	manFlagFile := fmt.Sprintf("%s/%s.1", manHelpPath, flagName)
-	manFlagFileAbs, err := filepath.Abs(manFlagFile)
+	// Read the embedded manual page
+	manFileName := fmt.Sprintf("docs/man/%s.1", flagName)
+	manContent, err := tracee.ManPagesBundle.ReadFile(manFileName)
 	if err != nil {
+		return errfmt.Errorf("manual page not found for %s: %v", flagName, err)
+	}
+
+	// Create a temporary file with the manual content
+	tmpFile, err := os.CreateTemp("", fmt.Sprintf("tracee-man-%s-*.1", flagName))
+	if err != nil {
+		return errfmt.WrapError(err)
+	}
+	defer func() {
+		_ = tmpFile.Close()
```

Comment:
The file is closed below, do we need to close it here again?

Reply:
the write below can fail before we explicitly close the file.
Calling close twice on the same file shouldn't be a problem